src/sdk/pynni/nni/__init__.py

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+__version__ = '999.0.0-developing'
```

Comment:
where this string is replaced with correct version?  this string in `setup.py` is replaced in `Makefile`